propra_webscience_ws24/training/main.py

```diff
@@ -128,13 +136,23 @@ def main(
     return _train_all_combinations(
         model_type_,
         training_combinations,
+        _parse_model_args(model_args),
         max_workers=max_workers,
     )
 
+def _parse_model_args(model_args: str) -> dict:
+    result = {}
+    if model_args is not None:
+        model_args = model_args.replace(" ", "").replace("=", ":")
+        for key_value in model_args.split(","):
+            kv = key_value.split(":")
+            result[kv[0]] = kv[1]
+    return result
```

Comment:
:thinking: Eventuell mit Guard-Clause etwas einfacher zu lesen?
```suggestion
    if model_args is not None:
        return {}

    model_args = model_args.replace(" ", "").replace("=", ":")
    result = {}
    for key_value in model_args.split(","):
        key, value = key_value.split(":")
        result[key] = value
    return result
```

Reply:
Hatte ich erst so ähnlich, hatte mir aber nicht gefallen. Kann es aber gerne mit Guard schreiben

Reply:
Kannst gerne so lassen, wie es ist. 
Alle meine Kommentare waren eigentlich nitpicks 😉.